components/script/dom/response.rs

```diff
@@ -248,11 +256,31 @@ impl ResponseMethods for Response {
         // TODO: This step relies on body and stream, which are still unimplemented.
 
         // Step 2
-        let response_response_clone = self.response.borrow().clone_for_dom_response();
         let new_response = Response::new(self.global().r());
-        *new_response.response.borrow_mut() = response_response_clone;
         new_response.Headers().set_guard(self.Headers().get_guard());
 
+        // https://fetch.spec.whatwg.org/#concept-response-clone
+        // This deviates from the spec a little, because there is no
```

Comment:
Not really "deviates from the spec". Instead, say "instead of storing a net_traits::Response internally, we only store the relevant fields, and only clone them here".